core/http/client/src/main/java/org/eclipse/rdf4j/http/client/SPARQLProtocolSession.java

```diff
@@ -1055,6 +1066,62 @@ protected HttpResponse execute(HttpUriRequest method) throws IOException, RDF4JE
 		}
 	}
 
+	static RDFFormat getContentTypeSerialisation(HttpResponse response) {
```

Comment:
Why is this a static? You're not using it in a static way, as far as I can tell, nor are there any other existing static methods in the ProtocolSession class. It think it can just be a private instance method.

Reply:
Doesn’t rely on any instance data. This means it can be static, and if it can be static it should be static. Also much easier to test when it’s static.